firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/FirebaseModelDownloader.java

```diff
@@ -463,11 +463,12 @@ private void deleteModelDetails(@NonNull String modelName) {
   }
 
   /**
-   * Update the settings which allow logging to firelog.
+   * Update the settings which allow logging to firelog. When not specifically set, defaults to use
+   * the Firebase wide data collection switch.
    *
    * @param enabled - is statistics logging enabled
    */
-  public void setStatsCollectionEnabled(boolean enabled) {
+  public void setMlDownloaderCollectionEnabled(boolean enabled) {
```

Comment:
Please add a java boolean over here. We also need to support the clear functionality to handle all edge cases.